tsfresh/feature_extraction/settings.py

```diff
@@ -152,7 +152,9 @@ def __init__(self):
             "lempel_ziv_complexity": [{"bins": x} for x in [2, 3, 5, 10, 100]],
             "fourier_entropy":  [{"bins": x} for x in [2, 3, 5, 10, 100]],
             "permutation_entropy":  [{"tau": 1, "dimension": x} for x in [3, 4, 5, 6, 7]],
-
+            # TODO: also the window parameter?
```

Comment:
I might actually leave the window parameter out, as that's something the power users can add in on their own.

Reply:
We could also do the same with "threshold" assuming we go the max subsequence route